nullaway/src/test/java/com/uber/nullaway/NullAwayAutoSuggestNoCastTest.java

```diff
@@ -55,12 +55,41 @@ public void setup() {
     flagsNoAutoFixSuppressionComment = b.build();
   }
 
+  // In EP 2.6.0 the newInstance() method we use below is deprecated.  We cannot currently address
+  // the warning since the replacement method was only added in EP 2.5.1, and we still want to
+  // support EP 2.4.0.  So, we suppress the warning for now
+  @SuppressWarnings("deprecation")
+  private BugCheckerRefactoringTestHelper makeTestHelper() {
+    return BugCheckerRefactoringTestHelper.newInstance(new NullAway(flags), getClass())
+        .setArgs(
+            "-d",
+            temporaryFolder.getRoot().getAbsolutePath(),
+            // the remaining args are not needed right now, but they will be necessary when we
+            // switch to the more modern newInstance() API
+            "-XepOpt:NullAway:AnnotatedPackages=com.uber,com.ubercab,io.reactivex",
+            "-XepOpt:NullAway:SuggestSuppressions=true",
+            "-XepOpt:NullAway:AutoFixSuppressionComment=PR #000000");
+  }
+
+  // In EP 2.6.0 the newInstance() method we use below is deprecated.  We cannot currently address
+  // the warning since the replacement method was only added in EP 2.5.1, and we still want to
+  // support EP 2.4.0.  So, we suppress the warning for now
+  @SuppressWarnings("deprecation")
+  private BugCheckerRefactoringTestHelper makeTestHelperNoSuppressionComment() {
```

Comment:
Almost feel, given the usages below, that this should be `makeTestHelper()` and the above one `makeTestHelperWithSuppressionComment()`. What do you think? (Perfectly fine to leave this for the future, though, but the most common method should have the shorter/unqualified name)

Reply:
4f1e47744e4322b7863e3ad07aa7ee7b08652984